SwiftSyft/Classes/SyftWebSocket_iOS12.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SyftWebSocket.swift
+//  SwiftSyft
+//
+//  Created by Sasha Bataieva on 25/01/2020.
+//
+
+import Foundation
+import Starscream
+
+public class SyftWebSocketIOS12: SyftWebSocket, WebSocketDelegate {
+    var socket: WebSocket!
+
+    public func didReceive(event: WebSocketEvent, client: WebSocket) {
+        switch event {
+        case .connected(let headers):
+            connected()
+            print("websocket is connected: \(headers)")
+        case .disconnected(let reason, let code):
+            disconnect()
+            print("websocket is disconnected: \(reason) with code: \(code)")
+        case .text(let string):
+            sendText(text: string)
+            print("Received text: \(string)")
+        case .binary(let data):
+            send(data: data)
```

Comment:
I think you're resending the text or data you receive from the socket. `WebSocketDelegate` from `Starscream` should handle received messages right? This should be `self.delegate.didReceive(socketMessage result: Result<Data, Error>)). For receiving text, you can just add another method to `SocketClientDelegate` to handle received text.

Reply:
fixed